guides-markdown/wallets.md

```diff
@@ -0,0 +1,357 @@
+
+While bcoin uses HD wallets, it does allow you to import individual keys into a wallet. This can be a handy feature in certain cases, but it means you'll need to backup any imported keys separately as they will not be recoverable simply by using your seed.
+
+But what exactly is the seed for an HD wallet? It can come in several forms, but bcoin implements [BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) which enables seeds to be represented by a mnemonic of twelve common words. This means your seed can be easily spoken, written down, or perhaps even memorized. But be careful! Your seed can be used to recover and spend everything in your HD wallet (except for the aforementioned imported keys), so treat it like you would an actual wallet with cash in it.
+
+### Accounts
+
+Wallets in bcoin are partitioned into accounts. When you first create a wallet, a "default" account is created automatically along with it. Accounts can be used to track and manage separate sets of keys all within a single wallet. For example, a business can use accounts to generate distinct addresses for depositors or to segregate customer funds internally. 
+
+Bcoin implements [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki) as a method of generating unlimited accounts deterministically. This adds additional dimensions to the heirarchy described above, meaning the same seed that can recover all your keys can also recover all your addresses.
+
+Each account also comes with its own "extended public key," a piece of data that can be used to generate all public keys for that account in a deterministic fashion. Remember that public keys can be used for receiving bitcoins, but not for spending. This means, for instance, that a business can create limitless deposit addresses for any number of its users without having to touch its critical private keys or seed.
+
+### Watch Only Wallets
+
+Speaking of not touching private keys, bcoin gives you the option to create wallets that are "watch only." Watch only wallets don't contain *any* private keys, which means they can't be used to spend the bitcoins they receive. However, they do a perfectly good job for creating addresses, receiving bitcoins, and detecting incoming transactions. Using watch only wallets where possible reduces the risk of your keys and bitcoin being stolen and is good security practice.
+
+Accounts always inherit the watch only behavior of their parent wallet. In other words, a watch only wallet will have exclusively watch only accounts while a non-watch only wallet will have only non-watch only accounts.
```

Comment:
It might be worth mentioning that an error is thrown if you try and import a private key into a watch only and also if you import a pubkey into a non-watch only (and then showing what the error is)